src/pages/previous_questions.js

```diff
@@ -29,26 +29,30 @@ function PreviousQuestions() {
   };
 
   return (
-    <div className="bg-previous min-h-screen bg-no-repeat bg-cover">
+    <div>
       <NavBar />
-      <div className="flex mt-8 ml-32 text-xl">
-        <h1 className="text-cerulean hover:underline">
-          {" "}
-          <a href="#/question">Weekly Question</a>
-        </h1>
-        <h1 className="text-white no-underline"> &nbsp;&gt; History</h1>
-      </div>
+      <div className="bg-previous min-h-screen bg-cover">
+        <div className="flex ml-32 text-xl">
+          <h1 className="text-cerulean hover:underline mt-8">
+            {" "}
```

Comment:
What's this for?

Reply:
Prettier added that for something that I had, but I don't anymore yea